app/models/person.rb

```diff
@@ -172,6 +172,10 @@ def self.search_order
     }.call
   end
 
+  def self.people_names(people)
```

Comment:
```
## TODO ----
# don't put presentation logic in the model!
```

Also, this is only used once, why don't you just move `.map(&:name).join(", ")` to where its used? Maybe the controller is also not the best place, but it's better than split this and add it to the model.